test/adapter/json_api/pagination_links_test.rb

```diff
@@ -174,6 +193,18 @@ def test_last_page_pagination_links_using_will_paginate
           assert_equal expected_response_with_last_page_pagination_links, adapter.serializable_hash
         end
 
+        def test_out_of_bounds_page_pagination_links_using_kaminari
+          adapter = load_adapter(using_kaminari(4), mock_request)
```

Comment:
might be good to replace `4` with `out_of_bounds_page_number = 4` here and below

Reply:
Done.